<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SavePlotDialog</class>
 <widget class="QDialog" name="SavePlotDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Save Plot</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0">
    <widget class="QLabel" name="scaleLabel">
     <property name="text">
      <string>Scale</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="qualityLabel">
     <property name="text">
      <string>Quality</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="fileTypeLabel">
     <property name="text">
      <string>File type</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="fileTypeBox"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="widthLabel">
     <property name="text">
      <string>Width</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="formatLabel">
     <property name="text">
      <string>Format</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="pdfTitleLabel">
     <property name="text">
      <string>PDF title</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="formatEdit">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="scaleEdit">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QLineEdit" name="pdfTitleEdit">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="2">
    <widget class="QCheckBox" name="costmeticCheckBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>Cosmetic</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="widthEdit"/>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="heightEdit"/>
   </item>
   <item row="9" column="1">
    <widget class="QPushButton" name="saveButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QPushButton" name="cancelButton">
     <property name="text">
      <string>Canel</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="heightLabel">
     <property name="text">
      <string>Height</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="qualityEdit">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="pdfCretorLabel">
     <property name="text">
      <string>PDF Creator</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLineEdit" name="pdfCreatorEdit">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
